src/components/FlightTable/FlightTable.tsx

```diff
@@ -1 +1,39 @@
 // component should go here
+import FlightRow from "../FlightRow/FlightRow";
+import * as React from "react";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import Paper from "@mui/material/Paper";
+import type { Flight } from "../../App";
+
+interface FlightTableProps {
+  flights: Flight[];
+}
+
+const FlightTable: React.FC<FlightTableProps> = ({ flights }) => (
```

Comment:
This is great! Small thing to note about using `React.FC<FlightTableProps>` as the type, it's generally okay but it will cause the `<FlightTable>` component to implicitly allow children which we might not want. For example, if you give a component the type `React.FC<FlightTableProps>` we can do this:

```tsx
<FlightTable>
     <div>Uh oh! This doesn't make sense!</div>
</FlightTable> 
```

It's not a huge deal, but we'll most likely use React's implicit return type of components which is done like this:
```diff
- const FlightTable: React.FC<FlightTableProps> = ({ flights }) => (
+ const FlightTable = ({ flights }: FlightTableProps) => (
```

An even more specific way of doing this would be to type the component as follows:
```diff
- const FlightTable: React.FC<FlightTableProps> = ({ flights }) => (
+ const FlightTable = ({ flights }: FlightTableProps): JSX.Element => (
```

All of these are correct, they just have different pros and cons!